enatega-multivendor-app/src/components/SubCategoryPage/SubCategoryPage.js

```
@@ -0,0 +1,867 @@
+    borderBottomWidth: 2,
+    borderBottomColor: 'transparent',
+    height: 35, // Reduce tab height
+    width: ITEM_WIDTH,
+    justifyContent: 'center' // Center text
+  },
+  selectedTab: { borderBottomColor: 'blue' },
+  categoryText: { fontSize: 14, fontWeight: 'bold' }, // Reduce font size
+  subcategoryContainer: {
+    height: 30, // Reduce height
+    flexDirection: 'row',
+    paddingHorizontal: 17,
```

Biome (1.9.4)

[error] This property value named paddingHorizontal is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named paddingHorizontal


(lint/suspicious/noDuplicateObjectKeys)